### DISCUSSION TITLE
Power Consumption a2dp

### DISCUSSION THREAD

**Discussion by pddc** [pschatzmann/arduino-audio-tools#639] [Q&A] [answered]
Phil, thanks for your work. Wanted to pick your brain on something.

We're trying to stream audio to I2S from an external A/D, to then send to an external speaker over BT, but have noticied that power consumption on the ESP would be too high.
What's your take on power consumption? specifically do you have any experience with running this on batteries (AAA, not lithium)

thanks

**Comment by pschatzmann** [comment, marked as answer]
I am sorry, but so far I never had any issues with the power. But to be honest because the wiring  was getting too tedious, so I am mainly using the AudioKit for my tests now...
